### ISSUE TITLE
Dependency issue for expo-file-system

### ISSUE THREAD

**Issue by fciannella** [victorsoares96/epubjs-react-native#112]
When installing on an expo manager app, I get this dependency issue:

I start by creating the app:

```
npx create-expo-app@latest AppName
```

I then cd into AppName and I install epubjs

```
npm install @epubjs-react-native/core

added 9 packages, changed 1 package, and audited 1231 packages in 2s

70 packages are looking for funding
  run `npm fund` for details

5 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```

Then I install the expo-file-system and I get this dep issue. I don't want to force it, I would like to know how to resolve it.

```
npx expo install @epubjs-react-native/expo-file-system
› Installing 1 other package using npm
> npm install --save @epubjs-react-native/expo-file-system
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: flashlit@1.0.0
npm ERR! Found: expo-file-system@15.4.2
npm ERR! node_modules/expo-file-system
npm ERR!   peer expo-file-system@"15.4.2" from @epubjs-react-native/core@1.3.0
npm ERR!   node_modules/@epubjs-react-native/core
npm ERR!     @epubjs-react-native/core@"^1.3.0" from the root project
npm ERR!   expo-file-system@"~15.4.0" from expo-asset@8.10.1
npm ERR!   node_modules/expo-asset
npm ERR!     expo-asset@"~8.10.1" from expo@49.0.11
npm ERR!     node_modules/expo
npm ERR!       peer expo@"*" from expo-application@5.3.1
npm ERR!       node_modules/expo-application
npm ERR!         expo-application@"~5.3.0" from expo@49.0.11
npm ERR!       6 more (expo-constants, expo-file-system, expo-font, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! @epubjs-react-native/expo-file-system@"*" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: expo-file-system@14.1.0
npm ERR! node_modules/expo-file-system
npm ERR!   peer expo-file-system@"14.1.0" from @epubjs-react-native/expo-file-system@1.0.0
npm ERR!   node_modules/@epubjs-react-native/expo-file-system
npm ERR!     @epubjs-react-native/expo-file-system@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /home/fciannella/.npm/_logs/2023-09-19T16_44_04_861Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /home/fciannella/.npm/_logs/2023-09-19T16_44_04_861Z-debug-0.log
Error: npm exited with non-zero code: 1
Error: npm exited with non-zero code: 1
    at ChildProcess.completionListener (/home/fciannella/src/expo/FlashLit/node_modules/@expo/spawn-async/build/spawnAsync.js:52:23)
    at Object.onceWrapper (node:events:628:26)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)
    ...
    at spawnAsync (/home/fciannella/src/expo/FlashLit/node_modules/@expo/spawn-async/build/spawnAsync.js:17:21)
    at NpmPackageManager.runAsync (/home/fciannella/src/expo/FlashLit/node_modules/@expo/package-manager/build/node/BasePackageManager.js:38:42)
    at /home/fciannella/src/expo/FlashLit/node_modules/@expo/package-manager/build/node/NpmPackageManager.js:39:20
    at /home/fciannella/src/expo/FlashLit/node_modules/@expo/package-manager/build/utils/spawn.js:19:34
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

How do I fix the dependency issue?

**Comment by ithustle** [comment]
@fciannella, did you solve this issue?

**Comment by fciannella** [comment]
Nope, I had to park it for some other priorities! Is there any guidance you
can give on how to fix it?

**Comment by ithustle** [comment]
@fciannella, I fixed 3h ago.
I had to install `expo-file-system` and `expo-modules-core` as dev dependencies.

**Comment by victorsoares96** [comment]
Hello everyone, thanks for reporting this problem to me and I'm sorry for the delay, the fix for this will be released in version 1.4.0 of the library!
